kubernetes/assets/dashboards/kubernetes_nodes.json

```
@@ -0,0 +1,1401 @@
+                        "order": "asc",
+                        "q": "avg:kubernetes_state.node.memory_allocatable{$cluster,$host} by {host}"
+                    }
+                ],
+                "viz": "query_table"
+            },
+            "title": true,
+            "title_align": "left",
+            "title_size": 16,
+            "title_text": "Allocatable Memory sort ASC",
+            "type": "query_table",
+            "width": 57,
+            "x": 0,
+            "y": 286
+        },
+        {
+            "height": 14,
+            "legend": false,
+            "tile_def": {
+                "custom_links": [],
+                "requests": [
+                    {
+                        "q": "sum:kubernetes_state.nodes.by_condition{$cluster,$host,condition:ready,status:true} by {condition,status,cluster_name}",
+                        "style": {
+                            "palette": "dog_classic",
+                            "type": "solid",
+                            "width": "normal"
+                        },
+                        "type": "line"
+                    }
+                ],
+                "viz": "timeseries",
+                "yaxis": {
+                    "includeZero": true,
+                    "label": "",
+                    "max": "auto",
+                    "min": "auto",
+                    "scale": "linear"
+                }
+            },
+            "title": true,
+            "title_align": "left",
+            "title_size": 16,
+            "title_text": "Nodes in Ready condition",
+            "type": "timeseries",
+            "width": 39,
+            "x": 0,
+            "y": 36
+        },
+        {
+            "height": 20,
+            "tile_def": {
+                "custom_links": [],
+                "requests": [
+                    {
+                        "conditional_formats": [
+                            {
+                                "comparator": ">",
+                                "palette": "white_on_green",
+                                "value": "3600"
+                            },
+                            {
+                                "comparator": ">",
+                                "palette": "white_on_yellow",
+                                "value": "0"
+                            },
+                            {
+                                "comparator": "<",
+                                "palette": "white_on_red",
+                                "value": "0"
+                            }
+                        ],
+                        "q": "top(max:tls.seconds_left{kubelet:server,$cluster,$host} by {host}, 25, 'last', 'asc')"
+                    }
+                ],
+                "viz": "toplist"
+            },
+            "title": true,
+            "title_align": "left",
+            "title_size": 16,
+            "title_text": "Kubelet certificate close to expirations",
```

Is there a title to phrase how we're comparing to `3600`? Some thing like:
```suggestion
            "title_text": "Kubelet certificates expirations (<1 hour left)",
```